template.yaml

```diff
@@ -136,8 +136,13 @@ Parameters:
       - "warn"
       - "error"
       - "debug"
+  PermissionBoundaryPolicyArn:
+    Description: ARN of a boundary policy if your organisation uses some for roles, optional.
+    Type: String
+    Default: ""
 
 Conditions:
+  CreateDefaultBoundaryPolicy: !Equals [!Ref PermissionBoundaryPolicyArn, ""]
```

Comment:
When I read this again it started to confuse me because the name is negated from what it does now.

Maybe we better use the condition name "ApplyPermissionsBoundary" and make it `!Not !Equals [!Ref PermissionBoundaryPolicyArn, ""]` (note the !Not)

Reply:
Sure, no default boundary is created anymore now. I'll amend.